src/components/canvas.test.tsx

```diff
@@ -0,0 +1,66 @@
+import * as Adapter from "enzyme-adapter-react-16";
+import * as React from "react";
+
+import { configure, mount, shallow } from "enzyme";
```

Comment:
Is `shallow` ever used?

Reply:
No, I was trying it out for speed vs mount and forgot to remove it.